spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/TaskDeleteService.java

```diff
@@ -53,6 +53,15 @@ public interface TaskDeleteService {
 	 */
 	void cleanupExecutions(Set<TaskExecutionControllerDeleteAction> actionsAsSet, String taskName, boolean completed);
 
+	/**
+	 * Clean up the resources that resuled from running the task with the given name and actions.
+	 *
+	 * @param actionsAsSet the actions
+	 * @param taskName the task name
+	 * @param completed the completion state of the task executions
```

Comment:
Missing the `@param days..` javadoc. Also, let's make it consistent w/ outcome of this comment.